web/packages/extension/src/content.ts

```diff
@@ -198,14 +198,18 @@ function isXMLDocument(): boolean {
             }
         }
     });
-
+    const autoStartOptions = options.autostart
+        ? {
+              autoplay: "on",
+              unmuteOverlay: "hidden",
+              splashScreen: false,
+          }
+        : {};
```

Comment:
The idea here, much like `explicitOptions`, is to only set these options if autostart is not the default (false), so if a self-hosted config sets autoplay to "on", for example, it's not ignored when the extension is used.

Reply:
This is not important if the self-hosted config takes precedence over the extension toggles as is currently the case.